content/docs/SUSHI/configuration/exhaustive-config.yaml

```diff
@@ -147,6 +163,10 @@ pages:
 # * useContext
 # * copyright
 # * packageId
+# The R5 ImplementationGuide resource also defines properties that
```

Comment:
I dunno.  Maybe it's just me.  But I feel like a blank line after the bulleted list is appropriate.
```suggestion

# The R5 ImplementationGuide resource also defines properties that
```